src/features/device-heartbeat/index.ts

```diff
@@ -150,6 +151,8 @@ interface MetricEventArgs {
 interface DeviceOnlineStateManagerMessage {
 	id: string;
 	currentState: DeviceOnlineStates;
+	/** The timestamp that the DB heartbeat was last updated */
+	updatedAt?: number;
```

Comment:
@Page- Should we shorten this property and/or change the timestamp to be storing seconds rather than ms?